docs/guides/connecting_to_your_data/cloud/s3/spark.md

```diff
@@ -2,4 +2,160 @@
 title: How to connect to your data on a S3 using spark
 ---
 
-**TODO** This is a stub
+import NextSteps from '../../components/next_steps.md'
+import Congratulations from '../../components/congratulations.md'
+import Prerequisites from '../../components/prerequisites.jsx'
+import WhereToRunCode from '../../components/where_to_run_code.md'
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+This guide will help you connect to your data stored on AWS S3 using spark.
+This will allow you to validate and explore your data.
+
+<Prerequisites>
+
+- Have access to data on an AWS S3 bucket
+- Have a access to a working spark installation
+
+</Prerequisites>
+
+## Steps
+
+### 1. Choose how to run the code in this guide
+
+<WhereToRunCode />
+
+### 2. `[🍏 CORE SKILL ICON]` Instantiate your project's DataContext
+
+Import these necessary packages and modules.
+
+```python file=../../../../../tests/integration/docusaurus/connecting_to_your_data/cloud/spark_s3_yaml_example.py#L1-L4
+```
+
+:::tip
+Load your DataContext into memory using one of the guides below based on your deployment:
```

Comment:
This may be a little too nit-picky, but what are your thoughts on : 

```suggestion
:::note  Load your DataContext into memory using one of the guides below based on your deployment:
```

Reply:
Because I dont think the instantiation of the DataContext would fall under a 'tip' but a requirement.